catalog/core/catalog-core-solr/src/main/java/ddf/catalog/source/solr/DynamicSchemaResolver.java

```diff
@@ -873,4 +882,21 @@ public String getSortKey(String field) {
   public Stream<String> anyTextFields() {
     return anyTextFieldsCache.stream();
   }
+
+  /**
+   * Gets the metadata size limit from custom.system.properties. Defaults to 5mb limit.
+   *
+   * @return Integer metadata size limit in bytes
+   */
+  private Integer getMetadataSizeLimit() {
+    String sizeLimit = System.getProperty("metadata.size.limit");
+    if (StringUtils.isNumeric(sizeLimit)) {
+      if (Integer.parseInt(sizeLimit) * 1000000 < Integer.MAX_VALUE) {
```

Comment:
❗️ Are you sure the value is 1000000 and not 1048576 (i.e. 1024 * 1024)

Reply:
Removed this. good catch though